cyfronet-fid/sat4envi
# [#278] static sentinel form styling

## What is in this PR?

* Sentinel Search form with stubbed data
* Loading for products / overlays
* Routing for products / sentinel search
* Akita update (fixing a few minor bugs)
* Modal Styling
* Fix saving `touched` state to form in HMR mode

### How to test?

Navigate to the main page click "Szukaj danych" the sentinel form should be visible


### How does it look like?

![image](https://user-images.githubusercontent.com/13235269/71898713-4b65c580-315a-11ea-898c-ec3c3cdd64f8.png)

![image](https://user-images.githubusercontent.com/13235269/71898537-e01bf380-3159-11ea-8ce9-6f125084b0df.png)


![image](https://user-images.githubusercontent.com/13235269/71898519-cda1ba00-3159-11ea-9106-9a7e3908df9d.png)

![image](https://user-images.githubusercontent.com/13235269/71898529-d5f9f500-3159-11ea-9747-a338a1301e35.png)
